bat-utils/lib/extras-hapi.js

```diff
@@ -154,7 +154,7 @@ const WreckProxy = (server, opts) => {
 
   if (!opts) opts = {}
   if (!opts.headers) opts.headers = {}
-  if (!opts.headers['user-agent']) opts.headers['user-agent'] = wreckUA
+  if (!opts.headers['user-agent'] && !opts.headers['User-Agent']) opts.headers['user-agent'] = wreckUA
```

Comment:
i do't object to the change, but what caused you to make the change? just curious.

Reply:
nvm... it's `go`

Reply:
I seemed to only be able to override the outgoing ip address by using `User-Agent` versus `user-agent` - but don't hold me to it :)